### ISSUE TITLE
Mer zoom in

### ISSUE THREAD

**Issue by lostminds** [josve/autotable#14]
Ändra så zoom (tryck z) zoomar in lite mer. Så blir det tydligare om en tittar på lite håll, och de nya mer högupplösta texturerna blir mer meningsfulla.

**Comment by local-minimum** [comment]
Tror zoomen styrs av detta https://github.com/josve/autotable/blob/master/src/game.ts#L146-L149 att det bara är att välja ett annat tal än 1, men kan mycket väl missa hur koden funkar och orkar inte sätta upp byggsystem på windows

**Comment by local-minimum** [comment]
själva zoomen sker sedan här https://github.com/josve/autotable/blob/master/src/main-view.ts#L145-L148 men det sätter bara avståndet mellan objektet och kameran och inte brännvidden, men det borde väl duga som zoom.

**Comment by lostminds** [comment]
Jo, jag har ju justerat lite https://github.com/josve/autotable/blob/master/src/main-view.ts#L137-L138 så kanske kan en justera mer. Men det är ju som sagt lite av ett projekt att sätta upp hela byggsystemet, det är bortom mitt kunnande. 

Så jag har kört det genom att bygga det direkt på jonas server via ssh helt mot alla principer, eftersom det ändå bara var jag som jobbade med det.

**Comment by local-minimum** [comment]
Tror mest på  att ändra här https://github.com/josve/autotable/blob/d0924fe7895b5d0d071b84cf481ea93d1325c747/src/main-view.ts#L142 för att vara påverka i zoomläge
